src/commands/user/level.ts

```diff
@@ -38,60 +30,58 @@ export default class Level {
     const selectedMember = guild.members.cache.filter((member) => member.user.id === id).map((user) => user)[0];
     const avatarURL = selectedMember.displayAvatarURL();
 
-    const levelEnabled = await settingsTable?.get(`${guild.id}.leveling.enabled`).then(
-      (data) => {
+    const levelEnabled = await settingsTable
+      ?.get(`${guild.id}.leveling.enabled`)
```

Comment:
```suggestion
    const levelEnabled = await settingsTable?
      .get(`${guild.id}.leveling.enabled`)
```
That character belongs there I think ;)